acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/SignTransactionAcceptanceTest.java

```diff
@@ -22,40 +22,109 @@
+
   @Test
   public void valueTransfer() throws IOException {
-    final String recipient = "0x1b00ba00ca00bb00aa00bc00be00ac00ca00da00";
     final BigInteger transferAmountWei = Convert.toWei("15.5", Unit.ETHER).toBigIntegerExact();
-    final BigInteger startBalance = ethNode().accounts().balance(recipient);
+    final BigInteger startBalance = ethNode().accounts().balance(RECIPIENT);
 
     final Transaction transaction =
         Transaction.createEtherTransaction(
             RICH_BENEFACTOR.address(),
             RICH_BENEFACTOR.getNextNonceAndIncrement(),
             GAS_PRICE,
             INTRINSIC_GAS,
-            recipient,
+            RECIPIENT,
             transferAmountWei);
 
     final String hash = ethSigner().transactions().submit(transaction);
 
     ethNode().transactions().awaitBlockContaining(hash);
 
     final BigInteger endBalance =
-        ethNode().accounts().balance(recipient, ethNode().transactions().blockContaining(hash));
+        ethNode().accounts().balance(RECIPIENT, ethNode().transactions().blockContaining(hash));
 
     assertThat(endBalance)
         .isCloseTo(startBalance.add(transferAmountWei), Offset.offset(BigInteger.ONE));
   }
 
+  @Test
+  public void multipleValueTransfers() throws IOException {
+    final BigInteger transferAmountWei = Convert.toWei("1", Unit.ETHER).toBigIntegerExact();
+    final BigInteger startBalance = ethNode().accounts().balance(RECIPIENT);
+
+    final Transaction transaction =
+        Transaction.createEtherTransaction(
+            RICH_BENEFACTOR.address(),
+            null,
+            GAS_PRICE,
+            INTRINSIC_GAS,
+            RECIPIENT,
+            transferAmountWei);
+
+    String hash = null;
+    for (int i = 0; i < NO_OF_TRANSACTIONS; i++) {
+      hash = ethSigner().transactions().submit(transaction);
+    }
+
+    ethNode().transactions().awaitBlockContaining(hash);
+
+    final BigInteger endBalance =
+        ethNode().accounts().balance(RECIPIENT, ethNode().transactions().blockContaining(hash));
+
+    BigInteger numberOfTransactions = BigInteger.valueOf(NO_OF_TRANSACTIONS);
+    assertThat(endBalance)
+        .isCloseTo(
+            startBalance.add(transferAmountWei.multiply(numberOfTransactions)),
+            Offset.offset(BigInteger.ONE));
+  }
+
+  @Test
+  public void valueTransferNonceTooLow() throws IOException {
+
+    valueTransfer(); // call this test to increment the nonce
+
+    final BigInteger transferAmountWei = Convert.toWei("15.5", Unit.ETHER).toBigIntegerExact();
+
+    final Transaction transaction =
+        Transaction.createEtherTransaction(
+            RICH_BENEFACTOR.address(),
+            RICH_BENEFACTOR
+                .getNextNonceAndIncrement()
+                .subtract(BigInteger.ONE), // same nonce as last tx
+            GAS_PRICE,
+            INTRINSIC_GAS,
+            RECIPIENT,
+            transferAmountWei);
+
+    try {
```

Comment:
Don't do it like this, I've added a method to `transactions` that takes care of this boiler plating

Reply:
Can't see that ...

Reply:
Now my PR is merged, the method is available.

Reply:
done